src/data/bedrock/block/convert/BlockStateDeserializerHelper.php

```diff
@@ -102,24 +106,24 @@ public static function decodeComparator(RedstoneComparator $block, BlockStateRea
 	}
 
 	/**
-	 * @phpstan-template TBlock of Copper|CopperSlab|CopperStairs
+	 * @phpstan-template TBlock of Copper|CopperSlab|CopperStairs|CopperBulb|CopperDoor|CopperTrapdoor|CopperGrate
```

Comment:
Maybe it's worth creating an abstract class or something like that to combine all kinds of copper blocks?

Reply:
Yeah, we could create an interface for that, I'm just not sure what name it would have

Reply:
Although this would really be work for another PR, because we need to edit other's blocks code.